wiki/Skin/en.md

```diff
@@ -1,10 +1,24 @@
----
-stub: true
----
-
 # Skin
 
-*See also: [Skinning](/wiki/Skinning)*\
-*For the dedicated sub-forum for skins, see [Skinning](https://osu.ppy.sh/community/forums/15)*
+A **skin** is a set of graphics (images) and custom sample sets (audio clips) that replace the default ones used by osu!. Many players use skins to allow for a more customised user experience. Most of the important UI elements and sounds in osu! can be "skinned". 
+
+## Reasons to Use Skins
+
+- Skins can be used to further optimize a player's gameplay, since many skins hide certain visual details or offer gameplay elements focused on a certain playstyle like specific mod combinations.
```

Comment:
```suggestion
- Skins can be used to further optimize gameplay, since many of them hide certain visual details or offer gameplay elements focused on a certain [playstyle](/wiki/Gameplay/Play_style), such as specific [mod](/wiki/Gameplay/Game_modifier) combinations.
```